stabilizer/src/main/java/com/hazelcast/stabilizer/coordinator/Coordinator.java

```diff
@@ -372,6 +374,29 @@ private void echo(String msg) {
         log.info(msg);
     }
 
+    private void uploadResourcesToWorkers() throws IOException {
+        if(RESOURCES_HOME == null){
```

Comment:
I think `RESOURCES_HOME` will always be `not null`. Have a look at `File.exists()` and `File.isDirectory()` to check if there is a valid directory path in `RESOURCES_HOME`.